NGitLab.Mock/Clients/MergeRequestClient.cs

```diff
@@ -377,6 +377,12 @@ public IEnumerable<Models.MergeRequest> FilterByQuery(IEnumerable<MergeRequest>
                     mergeRequests = mergeRequests.Where(mr => mr.Assignee?.Id == assigneeId);
                 }
 
+                if (query.ReviewerId != null)
+                {
+                    var reviewerId = string.Equals(query.ReviewerId.ToString(), "None", StringComparison.OrdinalIgnoreCase) ? (int?)null : int.Parse(query.ReviewerId.ToString());
```

Comment:
You missed any. If any => no need to filter

Reply:
I don't fully understand, what is Any or None here?

Reply:
Check the documentation: 
https://docs.gitlab.com/ee/api/merge_requests.html#list-merge-requests

```
Returns merge requests which have the user as a reviewer with the given user id. None returns merge requests with no reviewers. Any returns merge requests with any reviewer. Mutually exclusive with reviewer_username.
```